content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -133,7 +133,10 @@ function getResourcesItems(exhibition: UiExhibition): ExhibitionItem[] {
   });
 }
 
-function getAccessibilityItems(): ExhibitionItem[] {
+function getAccessibilityItems(exhibition: UiExhibition): ExhibitionItem[] {
+  const defaultAccessContent =
+    'Large-print guides, transcripts and magnifiers are available in the gallery';
```

Comment:
This is fine for now, although I wonder if at some point we should have a file with all these bits of microcopy in one place (a la `hardcoded-id.ts`), so they can be more easily reviewed by the content team.